README.md

```diff
@@ -1,223 +1,203 @@
 
-or alias `command(...)`
+**Running test cases using the tc argument**
 
+`> icetea --tc <test case name> --tcdir <test case search path>`
 
-```
-dut[number]             #dut index (1..)
-dut[string]             #dut nick name, or '*' to execute command in all duts
-cmd[string]             #command to be executed
-arguments[dictionary]   #optional argument list
-        wait = <boolean>           # whether retcode is expected before continue next command. True (default) or False
-        expected_retcode = <int>    # expected return code (default=0)
-        timeout=<int>              # timeout, if no retcode receive
-```
-
-## Documentation
-HTML documentation for Icetea can be built using sphinx. The source
-for the documentation is located in [doc-source](doc-source).
-For installation of sphinx see [installation](#installation).
+To run all existing test cases from the `examples` folder:
 
-A build script for the documentation is included in build_docs.py.
-It's a simple python scripts that also generates autodoc api documentation.
-Run the script with:
+`> icetea --tc all --tcdir examples`
 
-`python build_docs.py`
+**Running an example test case with hardware**
 
-Currently similar documentation is available in markdown format in
-[doc](doc). This documentation will be phased out and is not maintained
-as well as the sphinx documentation.
+In this example we assume that a compatible board has been connected
+to the computer and an application binary for said board is available.
 
-## Examples
+`> icetea --tc test_cmdline --tcdir examples --type hardware --bin <path to a binary>`
 
-**Note:** Following examples uses [`dummyDut`](test/dut/dummyDut.c)
-application. It works in unix systems.
-To run following examples please compile `dummyDut` first using make:
-`> make -C test/dut`.
+**Using metadata filters**
 
-To print available parameters:
+To run all test cases with testtype regression in the metadata:
 
-`> icetea --help`
+`> icetea --testtype regression --tcdir <test case search path>`
 
-To run a single test case with the process dut:
+The following metadata filters are available:
+* test type (--testtype)
+* test subtype (--subtype)
+* feature (--feature)
+* test case name (--tc)
+* tested component (--component)
+* test case folder (--group)
 
-`> icetea --tc test_cmdline --tcdir examples --type process --bin ./test/dut/dummyDut`
+For further details see our documentation linked
+at [the top](#icetea-test-framework) of this document.
 
-To run all existing test cases from the `examples` folder:
+**Running a premade suite**
+Icetea supports a simple suite file that describes a suite of test cases
+in json format.
 
-`> icetea --tc all --tcdir examples --type process --bin ./test/dut/dummyDut`
+`> icetea --suite <suite file name> --tcdir <test case search path> --suitedir <path to suite directory>`
 
-To debug dut locally with [GDB](https://www.gnu.org/software/gdb/):
+**Enabling debug level logging**
+Add -v or -vv to the command.
```

Comment:
maybe more description what -v and -vv does -> "-vv" gives more debugging loglines etc..